scripts/build_binary_and_layer_dockerized.sh

```diff
@@ -17,6 +17,15 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
+if [ -z "$AGENT_VERSION" ]; then
+    echo "No agent version set, agentVersionDefault will be used instead (for the metrics log intake endpoint)."
```

Comment:
What is the "metrics log intake endpoint"? Also I don't think you need to necessarily explain this in a log line.

Reply:
typo, just the metrics intake endpoint. `https://x-y-z-app.agent.datadoghq.com/api/beta/sketches?api_key=***`. Removed this whole warning bit